File: server/Models/Ingredient.cs
namespace all_spice_csharp.Models;

public class Ingredient : RepoItem<int>
{
  public string Name { get; set; }
  public string Quantity { get; set; }
  public int RecipeId { get; set; }
  public string CreatorId { get; set; }
  public Profile Creator { get; set; }
}

File: client/src/models/Ingredient.js
import { Account } from "./Account.js"




export class Ingredient{
  constructor(data){
    this.id = data.id
    this.creatorId = data.creatorId
    this.createdAt = new Date(data.createdAt)
    this.updatedAt = new Date(data.updatedAt)

    this.name = data.name
    this.quantity = data.quantity
    this.recipeId = data.recipeId
    // this.creator = data.creator? new Account(data.creator) : null
  }
}